### PR TITLE
data:Hrishi5111998

### PR DESCRIPTION
**Pull request by Hrishi5111998**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3416"></a>

### CODE REVIEW

File: data/Hrishi5111998.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+     "icon": "FaInstagram"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://linkedin.com/in/hrishikesh-tule",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/Hrish5111998",
```
Comment (ChinmayMhatre) [inline data/Hrishi5111998.json:25]:
```suggestion
      "url": "https://github.com/Hrishi5111998",
```
I believe you misspelled your username. please update.

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

File: data/Hrishi5111998.json (hunk 1 above)
Comment (kumarsonsoff3) [reply data/Hrishi5111998.json:25]: Great catch @ChinmayMhatre, love seeing your contribution to lots of PRs! 😍

Comment (Hrishi5111998) [reply data/Hrishi5111998.json:25]: > 

oh my bad I will update

Comment (ChinmayMhatre) [reply data/Hrishi5111998.json:25]: Thank you @kumarsonsoff3🙌. Great seeing you too.

File: data/Hrishi5111998.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "avatar": "https://github.com/Hrishi5111998.png",
+  "links":[
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/@hrishikesh7030",
+      "icon": "FaTwitter"
+    },
+    {
+     "name": "Instagram",
+     "url": "https://instagram.com/@hrishikesh2412",
```
Comment (kumarsonsoff3) [inline data/Hrishi5111998.json:15]: Please fix your Instagram URL. It should be something like this! 👇 🛠

```suggestion
     "url": "https://instagram.com/hrishikesh2412",
```

image

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Comment by Hrishi5111998** [comment]
> Looks good to merge. Thank you for your contribution! 🎉
> 
> 

thank you for merging request. Its feel great to contributing to this open source community.

**Pull request merged**